ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePageVulnerabilities.tsx

```diff
@@ -125,10 +122,11 @@ function ImagePageVulnerabilities({
             field: 'Severity',
             direction: 'desc',
         },
-        onSort: () => setPage(1),
+        onSort: () => setPage(1, 'replace'),
     });
 
-    const { data, previousData, loading, error } = useQuery<
+    // TODO Split metadata, counts, and vulnerabilities into separate queries
```

Comment:
Since all of the data on this page is loaded through a single query, a single change puts all components into a loading state.

e.g. Changing the sort on the data causes the summary cards to re-load, even though a sort will not change those results.

These queries should be split in the future for both backend performance reasons as well as better UX client side.